todo_app/tests_e2e/test_integration_e2e_selenium.py

```diff
@@ -1,43 +1,56 @@
 import time
 import os
 from threading import Thread
-from todo_app.data.trello_request_handler import real_trello_request_handler
 import pytest
 from dotenv import load_dotenv
 from todo_app import app
 import random
 import string
 import logging
+import pymongo
 
 @pytest.fixture(scope='module')
 def app_with_temp_board():
```

Comment:
This looks very similar to the setup in `test_integration_e2e` - could we commonise this setup in a test helper file, or the `__init__.py` file in this folder?